gpuallocator/allocator.go

```diff
@@ -6,7 +6,7 @@ import (
 	"fmt"
 	"runtime"
 
-	"github.com/NVIDIA/gpu-monitoring-tools/bindings/go/nvml"
+	"github.com/NVIDIA/go-gpuallocator/internal/gpulib"
```

Comment:
Note: if we use:
```suggestion
	nvml "github.com/NVIDIA/go-gpuallocator/internal/gpulib"
```
or even rename the internal package to `nvml` here it should highlight changes not related to this renaming below.

Reply:
@klueska Your point is to not commit the `internal/gpulib` here, but to merge the functionality in there into `go-nvlib/pkg/nvml` ? and remove from this PR